product-development/README.md

```diff
@@ -29,62 +29,25 @@ All Tilters are encouraged to collaborate in _definining_ high-level business in
-
-Dan and Nick will typically not be DRIs on epics, in order to encourage all Tilters to have more responsibility for innovation and execution of day to day work.
-
-## Picking a high priority unstarted epic
-When picking a high priority unstarted epic, look at the epics _above the line_ in the [backlog](https://app.clubhouse.io/windmill/epics?state_ids=500008016&state_ids=500008001) and choose a sensible one based on at least these example scenarios:
-- My skill set is better suited for the third highest priority epic, so I'll pick that one, and save the first two for other folks who can finish them more quickly.
-- The fourth highest priority epic looks to be small so I'll pick that one first and finish it as a quick win before the end of the week.
-- My teammate has expertise in the highest priority epic and I'd like to collaborate and learn from them. But they aren't free just yet. I'll work on the third highest priority epic for now and plan to come back to pair with my teammate next week.
-
-Dan B and Nick, and Victor are responsible for setting the location of the line.
-
-Victor is responsible for tracking [open customer requests](https://companies-b164c.firebaseapp.com/customer-requests) as an input for prioritization. 
-
-<img src="images/above-the-line.png" height="150" />
+## Prioritizing epics
 
-## Reducing scope
-A given epic may have a fairly broad initial business problem. The engineer(s) should carve out and define a smaller problem, and spec out a solution that can be accomplished within a target of **_7 business days_** (counted from when the epic moves into `In Progress`, more below on states). If the solution cannot be finished within 7 business days, keep reducing scope. The epic description should be updated with the smaller scope and acceptance criteria, and the engineer(s) should create additional epics and/or stories to capture the future work. Alternatively, the engineer(s) may create a separate epic with the smaller scope and start work on that one instead (putting the original epic back into the backlog).
+Dan B and Nick, and Victor are responsible for maintaining the priority of the backlog (higher priority toward the top in the Clubhouse view, when the triangle is pointing down).
 
-The target time period is business calendar time, regardless of how many folks are working on the epic. So if for unplanned circumstances a person in a two person epic needs to be pulled away, the remaining person (who should be the epic owner DRI) may reduce scope to try to meet the target time period due to reduced capacity, saving the unfinished work for a future epic.
+Before an epic is moved above the line:
+1. Its description should contain the [epic description template](#epic-description-template).
+2. The "Problem statement and references" section should be completed.
 
```

Comment:
Not strictly necessary to have in the handbook but if you wanna add some more context/intent, consider something like: "The goal is to make as clear as possible to the team _why_ an epic was prioritized, and to give an engineer starting an epic as much context/prior art as possible."

Reply:
took a stab. merging, but happy to make further changes!